bundle/edu.gemini.model.p1/src/main/xsd/Proposal.xsd

```diff
@@ -133,6 +134,7 @@
             <xsd:element name="texes"      type="TexesBlueprintChoice"/>
             <xsd:element name="trecs"      type="TrecsBlueprintChoice"/>
             <xsd:element name="visitor"    type="VisitorBlueprintChoice"/>
+	    <xsd:element name="alopeke"    type="AlopekeBlueprintChoice"/>
```

Comment:
alignment?

Reply:
Ooops. I'll fix that 😊

Reply:
Strange. It looked fine in emacs, but was off in IDEA.

Reply:
Looks like the stupid alignment is still off here... It looks fine on my computer. I'll fix that.

Reply:
Ahhh... it's an emacs thing. It's inserting a tab instead of spaces.